bower.json

```diff
@@ -29,13 +30,14 @@
     "paper-listbox": "PolymerElements/paper-listbox#2",
     "paper-tabs": "PolymerElements/paper-tabs#2",
     "paper-toast": "PolymerElements/paper-toast#2",
-    "promise-polyfill": "polymerlabs/promise-polyfill#^1.0.0",
-    "google-youtube": "GoogleWebComponents/google-youtube#^1.2.1",
-    "google-map": "GoogleWebComponents/google-map#^1.1.10",
-    "web-animations-js": "web-animations/web-animations-js#^2.2.5"
+    "promise-polyfill": "polymerlabs/promise-polyfill#2",
+    "google-youtube": "GoogleWebComponents/google-youtube#2",
+    "google-map": "GoogleWebComponents/google-map#2",
+    "web-animations-js": "web-animations/web-animations-js#^2.2.5",
+    "webcomponentsjs": "master"
   },
   "resolutions": {
-    "iron-selector": "2",
-    "webcomponentsjs": "^0.7.24"
+    "iron-location": "fix-ie-capture",
+    "webcomponentsjs": "master"
```

Comment:
but you're going to update this to depend on real releases before you merge this, right?